themes/survey/vanilla/views/subviews/privacy/privacy_text.twig

```diff
@@ -14,17 +14,48 @@
     Show the privacy message and the data security message will be available in a modal
 
 #}
-
-<!-- Privacy message -->
-<br/>
-<div class="row">
-    <div class="limit-text-window {{ aSurveyInfo.class.privacydatasectextbody }}">
-        {{ aSurveyInfo.datasecurity_notice }}
+<div class="privacy-bloc">
```

Comment:
just a small typo should be "privacy-block". Also the class is not used in any css, i assume its there for access to add customizations?

Reply:
Right for typo, and yes for adding customisation. 

Maybe i need to add again `{{ aSurveyInfo.class.privacydatasectextbody }}` ?